bugsnag-plugin-android-ndk/src/main/jni/utils/serializer.c

```diff
@@ -368,23 +434,23 @@ char *bsg_serialize_report_to_json_string(bugsnag_report *report) {
   JSON_Object *exception = json_value_get_object(ex_val);
   JSON_Value *stack_val = json_value_init_array();
   JSON_Array *stacktrace = json_value_get_array(stack_val);
-  json_object_set_value(event, "exceptions", exceptions_val);
```

Comment:
The string should remain as `"exceptions"` to be a valid payload. I therefore think it could make sense to name the 3rd param as `exceptions_val`, unless some other variable needed renaming?